cli/cmd/server_utils.go

```diff
@@ -414,3 +413,53 @@ func isUnhealthyOnlyDueToRAG(hr *HealthPayload) bool {
 	// 2. There are NO unhealthy non-RAG components
 	return hasUnhealthyRAG && !hasUnhealthyNonRAG
 }
+
+// convertToDockerPath converts Windows paths to Docker-compatible format
+func convertToDockerPath(hostPath string) string {
+	if runtime.GOOS == "windows" {
```

Comment:
Instead of doing a windows-specific check here, we should be able to use go's `filepath.FromSlash()` method, which takes an OS path and converts it to a "path" which uses forward slashes